### DISCUSSION TITLE
アップデートで正解表示を実装したい

### DISCUSSION THREAD

**Discussion by silli-3072** [lit-iphone-course/lit-iphone-course#83] [Question-original]
不正解選択時に、正解の選択肢のボタンを赤い枠線で囲むという実装をしたいです。
今、正解不正解の認識はボタンのテキストを用いており、そこからどのように実装すれば良いかわからなく、方法を教えていただきたいです。
GitHub→https://github.com/silli-3072/LA4_TimeKorean

**Comment by appleEmily** [comment]
正解か不正解を判別する→画面遷移ってなってると思うけど、
[自動で画面遷移]
画面遷移の前に1秒待つ(この間に正解が赤で囲われている。)っていう実装

[次って押したら画面遷移]
まず、nextボタンを作る
正解か不正解を判別する→ボタンを赤枠で囲う→nextボタンを押したら画面遷移

で、「正解不正解の認識はボタンのテキストを用いており」って書いてあるけど、そのcheckAnswerメソッドの中でどのボタンを囲うか指定してあげたら良いと思う！そのために、ボタンに投資番号をふる変数とかを用意してあげて〜ってやって行ったらできると思う！

ごめん、日本語がスーパーむずい可能性ある😂伝わりますか
